src/routes/compiler/compiler.ts

```diff
@@ -1,17 +1,17 @@
 import solc from 'solc';
```

Comment:
I don't see the second parameter of `compile` being ever passed in through this plugin.  The `input` param should already have all sources included.  We could vastly simplify this file by replacing everything with just the following.

I tested that this works with both Wizard (which does not pass in the second param) and Remix (which does its own compilation).
```js
import solc from 'solc';

import type { CompilerInput } from "$lib/models/solc";

export class SolidityCompiler { 
  compile(input: CompilerInput) {
    const output = solc.compile(JSON.stringify(input));
    return output;
  }
}
```